### ISSUE TITLE
Lunar craters are too exaggerated

### ISSUE THREAD

**Issue by 10110111** [Stellarium/stellarium#988]
Using Stellarium v0.19.1-86-g063f535f3b, I get the following image of the Moon. Compared with [the photo from Wikipedia](https://en.wikipedia.org/wiki/File:FullMoon2010.jpg) or [the one from NASA](https://solarsystem.nasa.gov/resources/2460/lunar-near-side/?category=moons_earths-moon) or [this one also from NASA](https://solarsystem.nasa.gov/resources/843/rare-full-moon-on-christmas-day/?category=moons_earths-moon), Stellarium's rendering has so much exaggerated craters in the south part of the Moon that the Tycho crater hides among the others, although it should be the most prominent on the full Moon, along with its [rays](https://www.popastro.com/moonwatch/moon_guide/geology7a.php), which are almost invisible in the Stellarium's image in the vicinity of Tycho.
![Screenshot - 040320 - 00:12:09](https://user-images.githubusercontent.com/6376882/75820376-4c5b5200-5dad-11ea-8877-691bf99daf97.png)

**Comment by 10110111** [comment]
Using NASA [CGI Moon Kit](https://svs.gsfc.nasa.gov/4720) (downloading the 2048×1024 TIFF version and converting to Stellarium's `moon.png`) I've achieved the following images (try opening them in adjacent tabs in your browser and switching between them to see the difference):

* Original Stellarium texture

![Screenshot - 040320 - 22:49:08](https://user-images.githubusercontent.com/6376882/75917148-99512e00-5e6a-11ea-8f2f-e99263947d41.png)
* NASA texture

![Screenshot - 040320 - 22:46:30](https://user-images.githubusercontent.com/6376882/75917170-a110d280-5e6a-11ea-8a3d-20d831034090.png)

The new one seems to be much more realistic. Need to check though, whether other phases are good enough. Maybe normal map or even the rendering shader also need changes (Moon's surface is not Lambertian, which the shader doesn't seem to take into account).

**Comment by gzotti** [comment]
On the contrary, Moon uses the Oren-Nayar shading model. But I agree that the rays of Tycho were not apparent enough. I cannot say currently what the source of the photo texture was. I think the normal map is quite OK. Just make sure the images match.

It is funny that some things look the same for a decade, and suddenly 4 people or so care for it.

**Comment by A-j-K** [comment]
This one quarter of four people only acknowledged there appeared to be an improvement. I don't particularly "care for it". :)

**Comment by gzotti** [comment]
I see a simple 1:1 exchange is not possible as the new map is much brighter and needs some finetuning. What's worse, the polar areas are much clearer in our current texture.

**Comment by 10110111** [comment]
Actually yes, I've now understood why the polar areas look strange (even in the texture itself): there were no data collected by the satellite, so these areas were somehow simulated. So this will need more work.

**Comment by Atque** [comment]
@gzotti, do people really care about the pole areas, though? They are not visible from Earth anyway. I think @10110111's texture is gorgeous (it looks really good in 1024x512 resolution too), and with a normal map, the lack of features near the poles is almost covered completely.

**Comment by 10110111** [comment]
OK, brightness can be fixed by correcting the NASA's gamma-overcorrection (they applied gamma of 2.8). The following command (effectively correcting to gamma 2.2=2.8*0.786) results in the image below:
```
convert ~/Downloads/lroc_color_poles_2k.tif -gamma 0.786 moon.png
```
![Screenshot - 060320 - 00:23:33](https://user-images.githubusercontent.com/6376882/76027060-0800bb00-5f41-11ea-960a-23620dd055d2.png)

@Atque 
> do people really care about the pole areas, though? They are not visible from Earth anyway.

Actually you shouldn't think that this affects only "poles": the whole ranges from 70° to 90° both north and south are missing. These can't be invisible unless you are very close to the Moon.

**Comment by gzotti** [comment]
It's just a bit too much. Softened/washed out poles at 80° would be less apparent and could be acceptable. 
It looks like the current texture is a mix with the elevation/displacement map. See e.g. the dark crater south of Tycho. Source reference is in the SUG.
Ok, the current texture is not perfect, but this proposed one is not globally better. There may be still better ones, or somebody just photoshop something together that looks globally better than the current texture.

**Comment by 10110111** [comment]
Might be worth trying to fetch some pixels from the maps and a video [here](http://lroc.sese.asu.edu/posts/707) (although the data used there are only for 643 nm). Still looking for less overcooked and more complete data...

**Comment by 10110111** [comment]
I've asked [a question at Space.StackExchange](https://space.stackexchange.com/q/41821/775) about where to find the complete reflectance map.

**Comment by Atque** [comment]
Solved by #1479?

**Comment by gzotti** [comment]
Yes, that one as well! I could not find it, thanks.
